src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -240,5 +240,39 @@ public void ActiveElement_Default_IsSupported()
 
             Assert.That(activeElement.Text, Is.EqualTo("Invoked!"));
         }
+
+        [TestCase(["ClassName", "TextBox"])]
+        [TestCase(["FrameworkId", "WPF"])]
+        [TestCase(["NonExistent", null])]
+        public void GetAttribute_TextBox_ReturnsValue(string attributeName, string expectedValue)
+        {
+            var driverOptions = FlaUIDriverOptions.TestApp();
+            using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
+            var element = driver.FindElement(ExtendedBy.AccessibilityId("TextBox"));
+
+            var value = element.GetDomAttribute(attributeName);
+
+            Assert.That(value, Is.EqualTo(expectedValue));
+        }
+
+        [Test]
+        public void GetAttribute_Recognises_Pattern_Property()
```

Comment:
To comply with `MethodUnderTest_Condition_ExpectedOutcome`, rename to:
```suggestion
        public void GetAttribute_PatternProperty_ReturnsValue()
```